### ISSUE TITLE
[enhancement]: Please consider bringing the Seed configuration to match other options by adding always visible switch

### ISSUE THREAD

**Issue by gramoflava** [invoke-ai/InvokeAI#2807]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What should this feature add?

The feature will raise visibility of the option and prevent myself (and I expect other users) from mistakenly fixating the seed.

When I use function "Use All" I am pleased that I can start right away, but if my Seed item is closed I often forget to check configuration and to enable randomizing the Seed. I presume that the problem why it wasn't done before is because it has a conflicting configuration (one must turn on Randomizing or to configure Seed parameters). So I prepared a mockup of a possible solution to that, by changing option from "Randomize Seed" to "Specify Seed".

### Aditional Content

![CleanShot 2023-02-25 at 13 18 16](https://user-images.githubusercontent.com/266328/221351988-2a7c6aac-6611-4f15-8523-9290b3cfacfd.png)

**Comment by psychedelicious** [comment]
Good idea! This trips me up all the time. We'll improve the UI to address this, probably as you have suggested.

**Comment by Void2258** [comment]
I agree with this. Seed might be important enough to move the general tab and have a "noise" options in other options. But at least should have a more visible toggle as this suggests.
The side panels should be rearranged in order of importance, for example Hig rez fix is currently hidden all the way at the bottom and is more important to have easy access to and to know if it is or is not on than Symmetry or upscaling.
